website/tests/test_leaderboard_scoring.py

```
@@ -0,0 +1,275 @@
+        with patch("website.leaderboard.transaction.atomic") as mock_atomic:
+            update_leaderboard_on_dsr_save(sender=DailyStatusReport, instance=dsr, created=True)
+            mock_atomic.assert_not_called()
+
+    def tearDown(self):
+        cache.clear()
+
+
+class UserProfileLeaderboardScoreTest(TestCase):
+    def setUp(self):
+        """Set up test data"""
+        self.user = User.objects.create_user(username="testuser", email="test@example.com", password="testpass123")
```

Ruff (0.14.7)

Possible hardcoded password assigned to argument: "password"

(S106)